reservation_units/utils/reservation_unit_reservation_scheduler.py

```diff
@@ -0,0 +1,140 @@
+import datetime
+
+import pytz
+from pytz import UTC
+
+from opening_hours.utils.opening_hours_client import OpeningHoursClient
+
+
+class ReservationUnitReservationScheduler:
+    MARCH = 3
+    APRIL = 4
+
+    def __init__(self, reservation_unit):
+        self.reservation_unit = reservation_unit
+
+        if self.reservation_unit.max_reservation_duration:
+            self.reservation_duration = (
+                self.reservation_unit.max_reservation_duration.total_seconds() / 3600
+            )
+        else:
+            self.reservation_duration = 1
+
+        self.start_time = datetime.datetime.now(tz=pytz.UTC) + datetime.timedelta(
```

Comment:
Hmm, what's happening with utz + 2 hours?

Reply:
yup, actually should be the default time zone :flushed: => I'll fix this